pkg/localstorev2/storer.go

```diff
@@ -108,6 +113,17 @@ type NetStore interface {
 	PusherFeed() <-chan *pusher.Op
 }
 
+type BinC struct {
```

Comment:
Some description would be great to have. The same goes for the `ReverseStore` below.